### DISCUSSION TITLE
Number of updates

### DISCUSSION THREAD

**Discussion by janmhamburg** [elementor/elementor#23872] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

There have been 4(!) Elementor updates in the last 10 days. We support almost 50 websites with Elementor. Since we have contractually guaranteed our customers that we will always keep the websites up to date, we had to constantly update almost 50 pages. It's good when errors are fixed and Elementor is great, but the gaps between releases are really annoying. Thank you.

### Describe the solution you'd like

Could you please try to test better before major releases and not use the customers as a testing team? Thank you!

**Comment by toltechinternet** [comment]
Hi @janmhamburg,

The Elementor team and members of the community try their best to identify and fix bugs for each release by going through the beta phase each time to address the feedback before a major release. While I do understand your frustration when these releases are released to the open public and there are issues, there is always a risk of abnormal defects occurring due to a number of reasons (such as third-party plugins, hosting environments, and custom code from developers on their clients websites).

Yes, there have been 4 released patches, but this demonstrates the team are quickly identifying and fixing these issues in a timely manner.

We are in a similar situation with having a contractual agreement with all of our clients that we will keep their websites up to date. We perform weekly maintenance checks of security, plugins, themes etc. There are free service providers (such as ManageWP or WPMUDEV Hub) that allow you to update all these plugins in bulk for all your clients' websites. This should remove the pain of constantly updating 50 pages because you have 50 websites. I would highly recommend if you are doing them in bulk to perform checks or have some sort of uptime monitoring in case a website breaks after rolling out the bulk updates.

I hope this helps :-)

**Comment by fabigrafo** [comment]
Don’t update every single time. If you wait 2 weeks, so you could update only one time!

**Reply by m1ga** [reply]
exactly. Unless you are facing one of the fixed issues of course. But like @fabigrafo says: just wait a week or two before updating your Elementor.

Otherwise: read the changelog and decide if you'll need to update. 

> ...before major releases

those are only minor releases (x.x.4). Nothing wrong with not updating all the time.

**Comment by tobias992** [comment]
Rule number 1 with Elementor updates:

Never update to x.x.0 version. Wait 2 weeks to get the bugfix release ;)
